Gemfile

```diff
@@ -51,6 +51,12 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.6'
   gem 'selenium-webdriver'
   gem 'shoulda-matchers', '~> 3.1'
+  gem 'vcr'
+  gem 'webmock', require: 'webmock/rspec'
+end
+
+group :test do
+  gem 'simplecov', require: false, group: :test
```

Comment:
`+group :test`
and `, group: :test`
a bit too much
anyway, you may remove this, as we have simplecov in other PR